content/docs/release-notes.md

```diff
@@ -2,6 +2,69 @@
 title: Release Notes
 ---
 
+## 2022-07-19
+
+### What's new
+
+* Add WAL offloading to s3 on safekeepers.
+Separate task is launched for each timeline and stopped when timeline doesn't
+need offloading.
+
+* Download WAL from S3 if it's not available in safekeeper dir (#1932)
```

Comment:
not sure what this means. is this is a followup bugfix for the wal S3 upload, i think we can drop this.